src/api/ledger/parse/account-order.js

```diff
@@ -1,14 +1,27 @@
 'use strict';
-const parseOrderBase = require('./order-base');
+const utils = require('./utils');
+const flags = utils.core.Remote.flags.offer;
+const parseAmount = require('./amount');
 
 // rippled 'account_offers' returns a different format for orders than 'tx'
```

Comment:
Yeah. It contains all the same information, but it even uses different values for the flags.